docker/build/Dockerfile-prod

```diff
@@ -1,25 +1,36 @@
 # Use the official Docker Hub Ubuntu 18.04 base image
 FROM ubuntu:18.04
+MAINTAINER Timesketch <timesketch-dev@googlegroups.com>
 
-# Setup install environment and Timesketch dependencies
-RUN apt-get update && apt-get -y install \
-    apt-transport-https \
-    apt-utils \
-    ca-certificates \
-    git \
-    libffi-dev \
-    lsb-release \
-    software-properties-common \
-    python3-dev \
-    python3-pip \
-    python3-psycopg2 \
- && rm -rf /var/lib/apt/lists/*
+# Create container with:
+# docker build --no-cache --build-arg PPA_TRACK="[dev|stable]"
+
+ARG PPA_TRACK=stable
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -y update
+RUN apt-get -y install apt-transport-https apt-utils
+RUN apt-get -y install libterm-readline-gnu-perl software-properties-common
+RUN add-apt-repository -y ppa:gift/$PPA_TRACK
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
```

Comment:
```suggestion

```

Doing an upgrade is considered a bit dangerous, because some packages cannot upgrade inside an unprivileged container. We trust that the Ubuntu image is being kept up2date.

Reply:
We've been using this with Plaso for a while now, and has not caused issues so far. The Dockerfile CI test should catch breakage early.

Reply:
For context: https://github.com/log2timeline/plaso/blob/master/config/docker/Dockerfile#L22

Reply:
Ack - lets keep it here then, and remove if we get problems later down the road.